uqcsbot/api.py

```diff
@@ -244,6 +255,13 @@ def get(self, name_or_id: str, default: T = None, use_cache: bool =True) -> Unio
     def __iter__(self):
         return iter(self._channels_by_id.values())
 
+    def _on_im_created(self, evt):
+        chan = evt['channel']
+
+        chan['name'] = chan['id']
+        chan['is_private'] = True
```

Comment:
At this stage I don't think it's necessary to set `is_private` to true, even if it is. We should leave it up to slack events to tell us if its private. Ultimately, if we want to implement a script that needs to decide if a channel is private or not, we can just do something like `if is_group or is_im:`. Look at my #261 PR and implement an `is_im` field similar to the new `is_group` :^)

Reply:
Yeah that sounds reasonable to me but I didn't want to change the channels or remove channel.name etc in case something relied on it